fms_core/models/_constants.py

```diff
@@ -3,3 +3,6 @@
 
 BARCODE_NAME_FIELD_LENGTH = 200
 TEMPORARY_RENAME_SUFFIX = "$"
+
+
+SAMPLE_KINDS = ['DNA', 'RNA', 'BLOOD', 'CELLS', 'EXPECTORATION', 'GARGLE', 'PLASMA', 'SALIVA', 'SWAB']
```

Comment:
This is wrong ... we are creating a table for sample kinds to get rid of constants. You can use constants for your migration but declare it inside the migration file instead. This will encourage use of the table instead of the constants.